### PR TITLE
docs: make english README the repository's default README

### PR DESCRIPTION
**Pull request by 413x1nkp**
some people might be scared off by a base README being in Chinese, so using a more universally known language like English seems to be more suited for open-source projects. i believe some people just come to the project, see the README and then they're frightened to even work on/with it

this change is curated by the #33 issue

### CODE REVIEW

File: docs/cn/README.cn.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+
+# Retrieval-based-Voice-Conversion-WebUI
+一个基于VITS的简单易用的变声框架<br><br>
+
+[![madewithlove](https://img.shields.io/badge/made_with-%E2%9D%A4-red?style=for-the-badge&labelColor=orange
+)](https://github.com/fumiama/Retrieval-based-Voice-Conversion-WebUI)
+
+![moe](https://counter.seku.su/cmoe?name=rvc&theme=r34)
+
+[![Licence](https://img.shields.io/badge/LICENSE-MIT-green.svg?style=for-the-badge)](https://github.com/fumiama/Retrieval-based-Voice-Conversion-WebUI/blob/main/LICENSE)
```
Comment (fumiama) [inline docs/cn/README.cn.md]: The Licence badges in all languages' readme can be replaced like EN.

**Review by fumiama — CHANGES_REQUESTED** [review]
Well done, and you can append another small change as below.

File: docs/cn/README.cn.md (hunk 1 above)
Comment (413x1nkp) [reply docs/cn/README.cn.md]: resolved in 3ff5713

File: docs/kr/README.ko.han.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ VITS基盤의 簡單하고使用하기 쉬운音聲變換틀
 
 ![moe](https://counter.seku.su/cmoe?name=rvc&theme=r34)
 
-[![Licence](https://img.shields.io/github/license/fumiama/Retrieval-based-Voice-Conversion-WebUI?style=for-the-badge)](https://github.com/fumiama/Retrieval-based-Voice-Conversion-WebUI/blob/main/LICENSE)
+[![Licence](https://img.shields.io/badge/LICENSE-MIT-green.svg?style=for-the-badge)](https://github.com/fumiama/Retrieval-based-Voice-Conversion-WebUI/blob/main/LICENSE)
```
Comment (fumiama) [inline docs/kr/README.ko.han.md]: OH, sorry, I mean replace the `https://img.shields.io/badge/LICENSE-MIT-green.svg?style=for-the-badge` to `https://img.shields.io/github/license/fumiama/Retrieval-based-Voice-Conversion-WebUI?style=for-the-badge` but seems they're already that url. 😂

What you need to replace are `fr` `ja` and `ko`.

**Review by fumiama — CHANGES_REQUESTED** [review]

File: docs/kr/README.ko.han.md (hunk 2 above)
Comment (413x1nkp) [reply docs/kr/README.ko.han.md]: Alrighty, will replace badges to repo's specific license

**Review by fumiama — APPROVED** [review]
Thanks!

**Pull request merged**